packages/mui-docs/src/NProgressBar/NProgressBar.js

```diff
@@ -16,75 +16,19 @@ NProgress.configure({
   `,
 });
 
-const styles = (theme) => {
-  if (!theme.nprogress.color) {
-    throw new Error(
-      'MUI: You need to provide a `theme.nprogress.color` property' +
-        ' for using the `NProgressBar` component.',
-    );
```

Comment:
I think there is no need to throw error if we use `palette.primary.main` from the theme.